.github/actions/setup-dev-certs/action.yml

```diff
@@ -0,0 +1,35 @@
+name: 'Setup Dev Certificates'
+description: 'Creates and trusts development certificates'
+
+inputs:
+  cert-filename:
+    description: 'Certificate filename'
+    required: false
+    default: 'geopilot.pem'
+  trusted-cert-filename:
+    description: 'Trusted certificate filename'
+    required: false
+    default: 'geopilot.crt'
+
+runs:
+  using: 'composite'
+  steps:
+      - name: Set certificate paths
+        shell: bash
+        run: |
+          echo "PEM_CERT_PATH=$(pwd)/certs/${{ inputs.cert-filename }}" >> $GITHUB_ENV
+          echo "UBUNTU_TRUSTED_CERT_PATH=/usr/local/share/ca-certificates/${{ inputs.trusted-cert-filename }}" >> $GITHUB_ENV
+      
+      - name: Create dev-certs
+        shell: bash
+        run: |
+          mkdir -p ./certs
+          dotnet dev-certs https --export-path ${{ env.PEM_CERT_PATH }} --no-password --format PEM
+          chmod -R 664 ./certs/*
```

Comment:
PP: Einheitliche Schreibweise verwenden. `$(pwd)/certs`, `./certs`
PP: Basis-Verzeichnis in Variable auslagern.